src/components/TableSortButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { TableSortProps } from 'models';
+
+const TableSortButton = ({
+    TableSort,
+    setTableSort,
+    nameSort,
+}: TableSortProps) => {
+    const setUserTableSortClick = () => {
+        const newValue =
```

Comment:
wow really too many ternaries here. consider moving this calculation to separate function + use `let` or switch` instead

Reply:
I used let with a readable if, since we have two conditions, which is not very suitable for switch.